TrueSDK/External/TCTrueSDK.m

```diff
@@ -57,6 +58,7 @@ - (void)setupWithAppKey:(nonnull NSString *)appKey
     self.appLink = appLink;
     self.titleType = TitleTypeDefault;
     self.locale = @"en_US";
+    self.urlScheme = [NSString stringWithFormat:@"%@-truesdk://", appKey];
```

Comment:
Is it required to add this global, also if it is required, make it private

Reply:
removed the global urlScheme property